subtests/docker_cli/create/create_tmpfs.py

```diff
@@ -0,0 +1,38 @@
+"""
+Create container with --tmpfs option and inspect it
+"""
+
+from create import create_base
+from dockertest.output import DockerVersion
+from dockertest.dockercmd import DockerCmd
+from dockertest.output import mustpass
+
+
+class create_tmpfs(create_base):
+
+    def initialize(self):
+        super(create_tmpfs, self).initialize()
+        # Minimum docker version 1.10 is required for --tmpfs
+        DockerVersion().require_server("1.10")
+
+    def run_once(self):
+        super(create_tmpfs, self).run_once()
```

Comment:
(unless you were planning to add more) There's no need to define a method that only calls the super-class method.  That happens automatic by **not** defining a method in the sub-class.